server/src/core/workspace/resolvers/workspace.resolver.ts

```diff
@@ -33,4 +34,21 @@ export class WorkspaceResolver {
       select: prismaSelect.value,
     } as Prisma.WorkspaceUpdateArgs);
   }
+
+  @Query(() => Workspace)
+  async currentWorkspace(
+    @AuthWorkspace() workspace: Workspace,
+    @PrismaSelector({ modelName: 'User' })
```

Comment:
@emilienchvt You're typing the selector and using the `User` model instead of `Workspace` ;)